concepts/05 Widgets/Slider/00 Overview.md

```diff
@@ -6,28 +6,249 @@ The **Slider** is a widget that allows an end user to set a numeric value on a c
 
 The following code adds a simple **Slider** to your page. The **min** and **max** options limit the range of accepted values. The **value** sets the initial value for the widget.
 
+---
+##### jQuery
+
     <!--HTML-->
     <div id="sliderContainer"></div>
 
     <!--JavaScript-->
     $(function() {
         $("#sliderContainer").dxSlider({
-            min: 0, max: 100,
+            min: 0,
+            max: 100,
             value: 25
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-slider
+        [min]="minValue"
+        [max]="maxValue"
+        [(value)]="value">
+    </dx-slider>
+
+    <!--TypeScript-->
+    import { DxSliderModule } from "devextreme-angular";
+    // ...
+    export class AppComponent {
+        minValue = 0;
+        maxValue = 100;
+        value = 25;
+    }
+    @NgModule({
+        imports: [
+            // ...
+            DxSliderModule
+        ],
+        // ...
+    })
+
+##### Vue
+
+    <template>
+        <DxSlider
+            :min="minValue"
+            :max="maxValue"
+            :value.sync="value"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxSlider } from 'devextreme-vue/slider';
+
+    export default {
+        components: {
+            DxSlider
+        },
+        data() {
+            return {
+                minValue: 0,
+                maxValue: 100,
+                value: 25
+            };
+        }
+    }
+    </script>
+
+##### React
+
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { Slider } from 'devextreme-react/slider';
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+
+            this.state = {
+                minValue: 0,
+                maxValue: 100,
+                value: 25
+            };
+
+            this.handleValueChange = this.handleValueChange.bind(this);
+        }
+
+        handleValueChange(e) {
+            const previousValue = e.previousValue;
+            const newValue = e.value;
+            // Event handling commands go here
+
+            this.setState({
+                value: newValue
+            });
+        }
+
+        render() {
+            return (
+                <Slider
+                    min={this.state.minValue}
+                    max={this.state.maxValue}
+                    value={this.state.value}
+                    onValueChanged={this.handleValueChange}
+                />
+            );
+        }
+    }
+
+    export default App;
+
+---
+
 In addition, you can specify the step of **Slider** values using the [step](/api-reference/10%20UI%20Widgets/dxSliderBase/1%20Configuration/step.md '/Documentation/ApiReference/UI_Widgets/dxSlider/Configuration/#step') option.
 
+---
+##### jQuery
+
+    <!--HTML-->
+    <div id="sliderContainer"></div>
```

Comment:
There's no need for this since we've already shown how to declare a container for the widget.

Reply:
fixed